src/TodoList.jsx

```diff
@@ -17,8 +18,8 @@ var todoList = [
 function TodoList () {
     return(
         <ul>
-        {todoList.map(function (item) {
-        return <li key={item.id}>{item.title}</li>;
+        {todoList.map(function(item){
+          return <TodoListItem title={item.title}/>;
```

Comment:
Make sure that any items inside a call to `.map()` have a `key` property on them. We took out the `key` property from the `li` when we moved it to a new component (not under the `.map` anymore) but we need to re-add it onto the `TodoListItem` here!

The browser console is currently throwing an error that signifies this:
```
Warning: Each child in a list should have a unique "key" prop.
```